apis/v1alpha2/httproute_types.go

```diff
@@ -147,9 +147,6 @@ type RouteTLSConfig struct {
 	// handshake for requests that match the hostname of the associated HTTPRoute.
 	// The referenced object MUST reside in the same namespace as HTTPRoute.
 	//
-	// This field is required when the TLS configuration mode of the associated
-	// Gateway listener is set to "Passthrough".
```

Comment:
Would this be accurate if "Passthrough" was replaced with "Terminate"?

Reply:
So `certificateRef` is a required field (syntactically), and IIUC you would need to set the `RouteTLSConfig` if the listener protocol is `HTTPS`, the mode is `Terminate`, you don't want the listener to use the default certificate and there is a TLS route override policy of `Allow`.

Phew! :joy:

Reply:
Yeah, that's a good point. This comment never really made sense, and a replacement one is not particularly straightforward or valuable, I'm fine with just removing this.